src/app/design-patterns/observer-rxjs/roulette.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface RouletteModel {
```

Comment:
Move the file path to `src/app/design-patterns/models` and change the name to `RoulettetBetModel`